### PR TITLE
Fixes incorrect regex parser + add test

### PR DESCRIPTION
**Pull request by illusional**

### CODE REVIEW

File: sample_metadata/parser/generic_parser.py [hunk 1]
```diff
@@ -67,7 +67,7 @@
     + VCF_EXTENSIONS
 )
 
-rmatch = re.compile(r'[_\.-][Rr]?[12]')
+rmatch = re.compile(r'[_\.-][Rr]?[12]\.fastq(\.gz)?$')
```
Comment (vladsaveliev) [inline sample_metadata/parser/generic_parser.py:70]: Should we care about `.fq.gz`?

Comment (illusional) [reply sample_metadata/parser/generic_parser.py]: Addressed in latest commit @vladsaveliev, could you please confirm?

Comment (vladsaveliev) [reply sample_metadata/parser/generic_parser.py]: Awesome, thank you!

File: scripts/fix_fastqs.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    """
+    Fix fastqs for all of MY projects
+    """
+    datasets = ProjectApi().get_my_projects()
+    for dataset in datasets:
+        await main(dataset)
+
+
+if __name__ == '__main__':
+    asyncio.new_event_loop().run_until_complete(for_all_projects())
```
Comment (vladsaveliev) [inline scripts/fix_fastqs.py:79]: Really enjoy your async code mastery. I should start using more of it too :)

**Review by vladsaveliev — APPROVED** [review]
Thank you for fixing it so quickly!

**Pull request merged**